html/elements/input/range.json

```
@@ -126,17 +126,26 @@
                     "notes": "Vertical orientation available via the `writing-mode` property (see [Creating vertical controls](https://developer.mozilla.org/docs/Web/CSS/CSS_writing_modes/Vertical_controls))."
                   },
                   {
-                    "version_added": "≤72",
+                    "version_added": "22",
```

Tested in BrowserStack Live, and in Firefox 22, the range input just looks like a regular text input, whereas in Firefox 23 it is supported, including vertical with `orient="vertical"`:

```suggestion
                    "version_added": "23",
```